docs/UserGuide.md

```diff
@@ -97,24 +97,22 @@ Format: `help`
 
 Adds a person to the address book.
 
-Format: `add n/NAME (p/PHONE_NUMBER | e/EMAIL) (r/MODULECODE[-ROLETYPE])+ [a/ADDRESS] [t/TAG]+`
+Format: `add n/NAME (p/PHONE_NUMBER | e/EMAIL) [r/MODULECODE[-ROLETYPE]]+ [a/ADDRESS] [t/TAG]+ [d/DESCRIPTION]`
 
 * `PHONE_NUMBER` is almost a free-form text field with minimal validation. Refer to the [input format section](#input-format) to find out more.
 * `MODULECODE` refers to a module code of a NUS module (e.g. CS1101S, MA1521)
 * `ROLETYPE` refers to one of the following: `student`, `ta`, `tutor`, `prof`, `professor`.
 * The `r/MODULECODE[-ROLETYPE]` parameter means that the person has the role for this module (e.g. `r/CS1101S-student` means that the person is a student of CS1101S).
 * In `r/MODULECODE[-ROLETYPE]`, `[-ROLETYPE]` is optional. In such cases, this means that the person is a student of that module (e.g `r/MA1521` means that the person is a student of MA1521).
 * If the same module is added multiple times, then it is assumed to be an error in user input, because a person should not have multiple roles (student, tutor, professor) at the same time (e.g. `r/CS1101S-student r/CS1101S-prof` is not allowed).
+* `ADDRESS` can take any values and can not be blank.
+* `TAG` can take any alphanumeric values and can not be blank.
+* `DESCRIPTION` can take any values.
 
 <box type="tip" seamless>
 
-**Tip:** A person must have at least one pair of module code and role type.
-</box>
-
-<box type="warning" seamless>
-
 **Duplicate Handling:**
-A person is considered a duplicate if another person in the address book has the same email address. The app will prevent adding contacts with duplicate emails. 
+A person is considered a duplicate if another person in the address book has the same email address or phone number. The app will prevent adding contacts with duplicate emails or phone numbers. 
```

Comment:
👍 Do you think we should mention the 2 checks only happen when the email or phone number of the person to be added is non-empty?

Reply:
I think should be fine, it is quite intuitive